neptune/new/attributes/atoms/artifact.py

```diff
@@ -53,7 +53,7 @@ def download(self, destination: str = None):
             file_destination.parent.mkdir(parents=True, exist_ok=True)
             driver.download_file(file_destination, file_definition)
 
-    def track_files_to_new(
+    def track_files(
```

Comment:
Why the siganture is different than `Handler::track_files`? Should be the same. And you haven't even changed implementation of handler. It tries to call inexisting methods. Do we have any tests for `Handler::track_files`?

Reply:
> Why the siganture is different than Handler::track_files? Should be the same.
IDK, it was implemented that way, I'll take closer look on that.

> And you haven't even changed implementation of handler. It tries to call inexisting methods.
Yes, my bad...

> Do we have any tests for Handler::track_files?
I guess not, since test passed. I'll take care of it.

Reply:
> Why the siganture is different than Handler::track_files? Should be the same.

It's not since we're getting `project_uuid` from `run` instance.

Reply:
Should be. Like it's said in our jira task - methods in Handler should be subset of methods from all types of Attributes.
Can we pass project_uuid to `Artifact.__init__`?

Reply:
> Can we pass project_uuid to Artifact.__init__?

It's not the best idea, but you were right, both `track_files` can have the same signature.

We do not have pass `project_uuid` at all, since `Artifact (attribute)` contains `run` object with that information.

Done.